io.github.mudkipworld.pngtuber-remix.yml

```diff
@@ -0,0 +1,127 @@
+        url: https://github.com/MudkipWorld/GDExtensionsPlayGround.git
+        commit: 5ea7f79120ed12f145b4bafe956ced2e10cbc8c3
+        dest: ./.deps/GDExtensionsPlayGround
+      - type: patch
+        path: 0000_PTR_save_to_user.patch
+      - type: patch
+        path: 0001_PTR_plugins.patch
+      - type: patch
+        path: 0002_PTR_version.patch
+      - type: patch
+        path: 0003_GI_fix.patch
+        dest: ./.deps/GI
+      - type: patch
+        path: 0004_CM_defaults.patch
+        dest: ./.deps/GDExtensionsPlayGround
+      - type: archive
+        url: https://github.com/godotengine/godot/releases/download/4.4.1-stable/Godot_v4.4.1-stable_linux.x86_64.zip
+        dest: ./godot
+        sha512: ef4e76880a514257175544952c61191106fdef3095b909bafed9fcbeb230c3e5533920a0f3012882dd4bbde83028a67549825794e2d2c3cf76eba7918b71370e
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://github.com/godotengine/godot/releases/download/4.4.1-stable/Godot_v4.4.1-stable_linux.arm64.zip
+        dest: ./godot
+        sha512: 8af79fe5fbc4cc42f5a7e4a52bd124973e084218f4d299cb5400595c6083ca800a59770c9c289b622dfc164fc3c6f369b362134ea2ac44223361657ea7941b09
+        only-arches:
+          - aarch64
+
+    build-commands:
+      # Create target
+      - |
+        mkdir -p /app/share/pngtuber-remix
+      # Compile GlobalInput extension from source
+      - |
+        cd .deps/GI/godot-cpp
+        scons target=template_release
+      - |
+        cd .deps/GI/libuiohook
+        mkdir build
+        cd build
+        cmake -S .. -D BUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=../dist
+        cmake --build . --parallel 2 --target install
+      - |
+        cd ./.deps/GI/src
+        rm -rf ./bin
+        scons build=release
+      - |
+        rm ./GI/linux/*.so
+        cp ./.deps/GI/src/bin/linux/* /app/bin/
+      # Compile godot-gif
+      - |
+        cd .deps/godot-gif
+        scons target=template_release
+      - |
+        rm ./godotgif/bin/*.so
+        cp ./.deps/godot-gif/demo/addons/godotgif/bin/libgodotgif.linux.* /app/bin/
+      # Compile CustomMeshDeform
+      - |
+        ln -s ../GI/godot-cpp .deps/GDExtensionsPlayGround/godot-cpp
+        cd .deps/GDExtensionsPlayGround/CustomMeshDeform
+        rm -rf ./bin
+        scons build=release
+      - |
+        rm ./CustomMesh/linux/*.so
+        cp ./.deps/GDExtensionsPlayGround/CustomMeshDeform/bin/linux/CustomMesh.linux.* /app/bin/
+      # Build
+      - |
+        git describe --tags --exclude nightly --always > "/app/share/pngtuber-remix/version.txt"
+        HOME=/app ./godot/Godot_v4.4.1-stable_linux.* --headless --path . --export-pack "Linux/X11" /app/share/pngtuber-remix/pngtuber-remix.pck
+      # Create runner
+      - |
+        cat > /app/bin/pngtuber-remix <<EOF
+        #!/usr/bin/sh
+        cd /app/share/pngtuber-remix
+        godot-runner --main-pack ./pngtuber-remix.pck
+        EOF
+        chmod +x /app/bin/pngtuber-remix
```

Comment:
don't inline shell scripts like this. if a startup shell script is needed to launch it, it should be upstream not in the manifest.

Reply:
I can fix it so that the script is not needed, I just have no preference on how to run godot things